src/docs/admin/AdminTutorials.adoc

```diff
@@ -0,0 +1,40 @@
+= Tutorials: ProActive Workflows & Scheduling
+include::../common-settings.adoc[]
+:toc-title: Tutorials for Administrators
+
+== Step by Step Tutorials
+
+=== Run As Me
+This is a step by step tutorial, which was designed and tested for Ubuntu 14.04.
+
+ProActive Workflows & Scheduling allows advanced user management with the “run as me” feature. Run as me allows
+to execute a task under the current ProActive user, if an equally named system user exists on the executing machine.
+
+*Following steps configure the Node with a sample user called Gord:*
+
+. Create a group called proactive `groupadd proactive’
+. Create a user running the ProActive Node, set it's primary group to proactive
+ ‘useradd -g proactive -G sudo -m proactive’
+. Set the password of proactive user ‘passwd proactive’
```

Comment:
Item 3, "Set the password of proactive user ‘passwd proactive’" should not it be Gord? or Gord should not be renamed proactive? or Gord is considered as a ProActive specific user ? In this last case passwd command should be `passwd Gord`, no?